types/opossum/opossum-tests.ts

```diff
@@ -132,3 +132,18 @@ window[0].fires; // $ExpectType number
 const noTimeoutOptions: CircuitBreaker.Options = {
     timeout: false, // false value deactivate timeout
 };
+
+// you can call with a provided this arg
+const context = {test: 'test' as const};
+async function proxyFn(this: typeof context, ...args: number[]) {
+    return {
+        args,
+        thisArg: this.test,
+    };
+}
+const args = [1, 2, 3];
```

Comment:
```suggestion
const args: number[] = [1, 2, 3];
```
Might be a bit more future proof in case in the future typescript defaults this to a const version, although not sure if they would ever actually do that 🤷